SportsHub/SportsHub.Core/PasswordHasher/IPasswordHasher.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SportsHub.Domain.PasswordHasher
+{
+    public interface IPasswordHasher
+    {
+        string Hash(string password);
+
+        (bool Verified, bool NeedsUpgrade) Check(string hash, string password);
```

Comment:
I'm not a fan of Tuple as a return type. As for me, I would create a class something like `PasswordCheckResult` with 2 properties and return it. But you can leave it for now.

Reply:
Implemented PasswordCheckResult